test/featureflags.go

```diff
@@ -32,3 +34,17 @@ func requireAnyGate(gates map[string]string) func(context.Context, *testing.T, *
 		t.Skipf("No feature flag matching %s", strings.Join(pairs, " or "))
 	}
 }
+
+// GetEmbeddedStatusValue gets the current value for the "embedded-status" feature flag.
+// If the flag is not set, it returns the default value.
+func GetEmbeddedStatusValue(ctx context.Context, t *testing.T, kubeClient kubernetes.Interface) string {
```

Comment:
super nit: this function can just be called "getEmbeddedStatus"

Reply:
👍